package.json

```diff
@@ -177,6 +177,9 @@
       "src/util/testUtils.tsx",
       "src/util/updateSitemap.ts",
       "src/util/date-fns/locale/buildLocalizeFn.ts"
+    ],
+    "transformIgnorePatterns": [
+      "<rootDir>/node_modules/(?!hds-react)/"
```

Comment:
Did only Navigation component have the problmen that required this? strange

Reply:
yes, only...

Reply:
I think the reason is that other HDS component are imported from `"hds-react"` not from `"hds-react/components...`

Reply:
That would explain. Can you change the import?

Reply:
@nenonja

Reply:
Tested this and changing the import style fixes the problem and this can be deleted

Reply:
Yes that did the trick: https://github.com/City-of-Helsinki/events-helsinki-ui/pull/147/commits/c8310040a6fb7dddc34015b6ddc2e67761a7fc32